playwright.config.ts

```diff
@@ -0,0 +1,107 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+import { devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+	testDir: './src/ui/tests',
+	/* Maximum time one test can run for. */
+	timeout: 30 * 1000,
+	expect: {
+		/**
+		 * Maximum time expect() should wait for the condition to be met.
+		 * For example in `await expect(locator).toHaveText();`
+		 */
+		timeout: 5000
+	},
+	/* Run tests in files in parallel */
+	fullyParallel: true,
```

Comment:
This will be a problem since we will have to hit a real cluster for each test and those cant be run on top of each other.

Reply:
https://playwright.dev/docs/test-parallel#disable-parallelism we can disable globally, and re-enable on a per test file basis to speed up tests we know can be parallelized